README.md

```diff
@@ -1,11 +1,5 @@
-[![Banner](https://i.imgur.com/2bQmNZB.png)](https://github.com/SpaceEnergy/SpaceTheme-Discord)
+# SpaceTheme for Steam
 
-# SpaceTheme for Steam | v1.0.4-alpha
-[Join Discord Server for Support, News and Updates](https://discord.gg/7Zv8Xz3Vzn)
-
-[![ko-fi](https://ko-fi.com/img/githubbutton_sm.svg)](https://ko-fi.com/spacetheme)
-
-## Preview v1.0.4-alpha
 ![Preview](https://i.imgur.com/YYSSkBy.png)
```

Comment:
change alt text to "v1.0.4-alpha preview"

Reply:
merge the pull request and make further changes you see fit.